boards/shields/nrf7002ek/doc/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _nrf7002ek:
+
+nRF7002 EK
+##########
+
+Overview
+********
+
+The nRF7002 EK is a versatile evaluation kit in the form of an Arduino shield that can be used in Nordic and non-Nordic host boards.
+The nRF7002 EK unlocks low-power Wi-Fi 6 capabilities for your host device.
+Seamlessly connect to Wi-Fi networks and leverage Wi-Fi-based locationing, enabling advanced features such as SSID sniffing of local Wi-Fi hubs
+
+.. figure:: nrf7002ek.png
+   :alt: nRF7002 EK
+   :align: center
+
+   nRF7002 EK
+
+Requirements
+************
+
+The nRF7002 EK board is designed to fit straight into an Arduino connector and uses SPI as the communication interface.
+Any host board that supports the Arduino connector can be used with the nRF7002 EK.
```

Comment:
"The line length is 100 columns or fewer. In the documentation, longer lines for URL references are an allowed exception."

Reply:
Thanks fixed, will wait for any further comments before pushing.